computing_provider/computing_worker/tasks/build_space.py

```diff
@@ -27,11 +27,11 @@ def delete_space_task(self, space_name):
 
 
 @celery_app.task(name=BUILD_SPACE_TASK, bind=True)
-def build_space_task(self, space_name):
+def build_space_task(self, space_name, wallet_address):
     logger.info(
-        f"Attempting to download spaces from Lagrange. spaces name:{space_name}"
+        f"Attempting to download spaces from Lagrange. spaces name and wallet: {space_name}, {wallet_address}"
     )
-    space_api_response = requests.get(f"http://api.lagrangedao.org/spaces/{space_name}")
+    space_api_response = requests.get(f"http://api.lagrangedao.org/spaces/{wallet_address}/{space_name}")
```

Comment:
this part is specific on prod env , it should compatible both local and prod

Reply:
updated